BUCK

```diff
@@ -19,3 +19,8 @@ export_file(
     name = "rustfmt.toml",
     visibility = ["PUBLIC"],
 )
+
+build_workspace_node_modules(
```

Comment:
I don't think this will solve it as it is downloading them into a folder under `buck-out`. I think we need a `dev_` version of this to solve the issue at hand... or?

Reply:
I thought so at first when I saw the si implementation, but it works and creates all the node_modules folders locally as well. I believe they did it this way so that buck co-ordinates when this actually gets executed and can co-ordinate its behavior as a dependency and for concurrent jobs

Reply:
do you understand why it creates the node_modules locally as well? Can you show me?